homeassistant/components/comelit/strings.json

```diff
@@ -31,5 +31,24 @@
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "zone_status": {
+        "state": {
+          "alarm": "Alarm",
+          "armed": "Armed",
+          "open": "Open",
+          "excluded": "Excluded",
+          "faulty": "Faulty",
+          "inhibited": "Inhibited",
+          "isolated": "Isolated",
+          "rest": "Rest",
+          "sabotated": "Sabotated",
+          "unavailable": "Unavailable",
+          "unknown": "Unknown"
```

Comment:
Please return None if the value of the entity is unknown

Reply:
So you prefer to change the following logic:

```
    @property
    def native_value(self) -> StateType:
        """Sensor value."""
        zone: ComelitVedoZoneObject = self.coordinator.data[ALARM_ZONES][
            self._zone.index
        ]
        return zone.human_status.value
```

with a more complex one just for that ?

Reply:
Yes, because otherwise the automation editor will show 2 times Unknown, and stuff like the recorder can handle it better

Reply:
Done